plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/datatools/ui/wizard/DriverDefinitionWizard.java

```diff
@@ -54,10 +52,9 @@ public void create(DriverDefinition driverDefinition) {
 	 * @see org.jboss.reddeer.jface.wizard.WizardDialog#finish()
 	 */
 	@Override
-	public void finish() {		
-		new DefaultShell(WIZARD_TITLE);
```

Comment:
I would keep this line. Not sure what user is doing with wizard and maybe it's necessary to keep it here (e.g. to get focus on the shell).